codegen/transforms/svc_types.sysl

```diff
@@ -26,7 +26,7 @@ CodeGenTransform:
           "STRING_8" => "string"
           "BOOL" => "bool"
           "DATE" => "date.Date"
-          "DATETIME" => "convert.JSONTime"
+          "DATETIME" => if "time_format" in t.attrs then if "stdtime" == t.attrs.time_format then "time.Time" else "convert.JSONTime" else "convert.JSONTime"
```

Comment:
Use && to combine the first two conditions if "time_format" in t.attrs then if "stdtime" == t.attrs.time_format.

Reply:
I cant use && because the && operator doesnt short circuit in sysl. So using && will result in a runtime error.

Reply:
I think it should, please open an issue in sysl repo, if it doesn't.